src/gui/log/logmodel.cpp

```diff
@@ -142,10 +142,10 @@ LogMessageModel::LogMessageModel(QObject *parent)
     : BaseLogModel(parent)
     , m_foregroundForMessageTypes
     {
-        {Log::NORMAL, UIThemeManager::instance()->getColor(QLatin1String("Log.Normal"), QApplication::palette().color(QPalette::WindowText))},
-        {Log::INFO, UIThemeManager::instance()->getColor(QLatin1String("Log.Info"), Qt::blue)},
-        {Log::WARNING, UIThemeManager::instance()->getColor(QLatin1String("Log.Warning"), QColor {255, 165, 0})}, // orange
-        {Log::CRITICAL, UIThemeManager::instance()->getColor(QLatin1String("Log.Critical"), Qt::red)}
+        {Log::NORMAL, UIThemeManager::instance()->getColor(QLatin1String("Log.Normal"), QColorConstants::Svg::mediumseagreen)},
```

Comment:
Please put screenshot in the first post if the color really changed.

Reply:
Okay. I'll grab screenshots on Ubuntu and Windows

Reply:
>Please put screenshot in the first post if the color really changed.

There was screenshots from windows posted previously in https://github.com/qbittorrent/qBittorrent/pull/15404#issuecomment-1028932451